README.md

```diff
@@ -1,20 +1,59 @@
+
+# MochiWeb
 MochiWeb is an Erlang library for building lightweight HTTP servers.
 
-The latest version of MochiWeb is available at https://github.com/mochi/mochiweb
+## Overview
+MochiWeb provides a lightweight and fast solution for building HTTP servers in Erlang. The library is designed to be easy to use and provides a comprehensive set of features for building robust and scalable HTTP servers.
```

Comment:
I'm not sure I'd make the claim that it's comprehensive at this point, without support for HTTP/2 and likely other standards that came about in the past decade. Easy to use is also a bit questionable, since there was never a concerted effort to write good documentation for it.